### PR TITLE
Init shape analysis filecheck

### PR DESCRIPTION
**Pull request by tanyokwok**
As discussed in https://github.com/alibaba/BladeDISC/pull/707, this pull request initialize the shape analysis file check configurations.

Another issue is that it seems the `shape_analysis_tests.cpp` added in https://github.com/alibaba/BladeDISC/pull/707 is not compiled and run. cc @Yancey1989

### CODE REVIEW

File: pytorch_blade/tests/torchscript/shape_analysis.graph [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+// CHECK-LABEL: graph
+graph(%p1 : Float(*, *, *, requires_grad=0),
+      %p2 : Float(*, *, *, requires_grad=0),
+      %p3 : int):
+// CHECK: Float(*, *, *) = aten::add(%p1, %p2, %p3)
+  %1 : Tensor = aten::add(%p1, %p2, %p3)
+  return (%1)
+
+// CHECK-LABEL: graph
+graph(%p1 : Float(*, *, requires_grad=0, device=cpu)):
```
Comment (Yancey1989) [inline pytorch_blade/tests/torchscript/shape_analysis.graph:28]: Need to configure `stride` ? ShapeAnalysisPass uses [isComplete()](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/core/jit_type.h#L285) to check static shape or not.

Comment (tanyokwok) [reply pytorch_blade/tests/torchscript/shape_analysis.graph:28]: OK. I know why the concrete shapes can't be propagated now. IMO, our shape analysis pass will not depends on strides setting.

Comment (tanyokwok) [reply pytorch_blade/tests/torchscript/shape_analysis.graph:28]: And we should fix it in another PR.

File: pytorch_blade/tests/torchscript/shape_analysis_tool.cpp [hunk 2]
```diff
@@ -0,0 +1,80 @@
+// Copyright 2022 The BladeDISC Authors. All rights reserved.
```
Comment (Yancey1989) [inline pytorch_blade/tests/torchscript/shape_analysis_tool.cpp:1]: This filename is too general, maybe `file_checker` is better? And pass the `shap_analysis_pass` as argument.

Comment (tanyokwok) [reply pytorch_blade/tests/torchscript/shape_analysis_tool.cpp:1]: This is not a file checker. It just read in graph, do shape analysis, and output the graph again. FileCheck is handler with llvm lit.

File: pytorch_blade/pytorch_blade/compiler/jit/BUILD [hunk 3]
```diff
@@ -69,9 +39,25 @@ cc_library(
...
+        "//pytorch_blade/common_utils:torch_blade_common",
+        "@local_org_torch//:libtorch",
+    ],
+    alwayslink = True,
+)
+
 filegroup(
     name = "torch_blade_jit_test_srcs",
-    srcs = glob(["**/*_test.cpp"]),
+    srcs = glob(["*_test.cpp"]),
```
Comment (Yancey1989) [inline pytorch_blade/pytorch_blade/compiler/jit/BUILD:60]: this would exclude `torch/**_test.cpp` ?

**Review by Yancey1989 — APPROVED** [review]
LGTM

**Comment by tanyokwok** [comment]
mark related to https://github.com/alibaba/BladeDISC/issues/766

**Pull request merged**